doc/nrf/protocols/matter/getting_started/adding_clusters.rst

```diff
@@ -159,6 +159,15 @@ To edit clusters using the ZAP tool, complete the following steps:
 
 At this point, new clusters have been added to the Matter device.
 
+.. note::
+   On the first run the ZAP tool creates :file:`.zap` directory to store cached information for the following runs.
+   The default directory location is User's Home directory and it can be overriden by adding ``--stateDirectory`` and location path to the invoked ZAP commands.
```

Comment:
```suggestion
   The default directory location is the user's home directory and it can be overridden by adding ``--stateDirectory`` and the location path to the invoked ZAP commands.
```